conf/jetty/jetty.xml.production

```diff
@@ -827,8 +827,8 @@
         <New class="org.eclipse.jetty.util.resource.ResourceCollection">
           <Arg>
             <Array type="String">
-              <Item><SystemProperty name="jetty.zimlet.base" default="."/>/webapps/zimlet</Item>
-              <Item><SystemProperty name="jetty.zimlet.base" default="."/>/../zimlets-deployed</Item>
+		    <Item><SystemProperty name="jetty.base" default="."/>/webapps/zimlet</Item>
+		    <Item>./../zimlets-deployed</Item>
```

Comment:
can we try one more approach?
we can pass zimbra.base param from zmmailboxdmgr which points to /opt/zimbra and then here we can use it like
`<Item><SystemProperty name="zimbra.base" default="."/>/zimlets-deployed</Item>`

above solution gives more clarity then the current solution